web/src/features/fireWeather/components/NoonWeatherValueTable.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+
+import { ModelValue } from 'api/modelAPI'
+import { NoonForecastValue } from 'api/forecastAPI'
+import { MODEL_VALUE_DECIMAL } from 'utils/constants'
+import SortableTableByDatetime, {
+  Column
+} from 'features/fireWeather/components/SortableTableByDatetime'
+
+/**
+ * Reusable component used to display noon forecasts (issued by forecasters)
+ * and/or model forecasts (generated by computer-run models) in table format
+ */
+
+type Id =
+  | 'datetime'
+  | 'temperature'
+  | 'relative_humidity'
+  | 'wind_direction'
+  | 'wind_speed'
+  | 'total_precipitation'
+
+const columns: Column[] = [
+  {
+    id: 'datetime',
+    label: 'Date (PDT)',
+    align: 'left',
+    formatDt: (value: string) => value.slice(0, 10)
+  },
+  {
+    id: 'temperature',
+    label: 'Temp (°C)',
+    align: 'right',
+    format: (value: number) => value.toFixed(MODEL_VALUE_DECIMAL)
+  },
+  {
+    id: 'relative_humidity',
+    label: 'RH (%)',
+    align: 'right',
+    format: (value: number) => Math.round(value)
+  },
+  {
+    id: 'wind_direction',
+    label: 'Wind Dir',
+    align: 'right',
+    format: (value: number) => Math.round(value)
+  },
+  {
+    id: 'wind_speed',
+    label: 'Wind Spd (km/h)',
+    minWidth: 70,
+    maxWidth: 100,
+    align: 'right',
+    format: (value: number) => value.toFixed(MODEL_VALUE_DECIMAL)
+  },
+  {
+    id: 'total_precipitation',
+    label: 'Precip (mm/cm)',
+    minWidth: 70,
+    maxWidth: 100,
+    align: 'right',
+    format: (value: number) => value.toFixed(MODEL_VALUE_DECIMAL)
```

Comment:
the precip values aren't being displayed in the noon GDPS values table in your PR's route

Reply:
Ah shoot we don't use total_precipitation property for models anymore